macros/LearnSidebar.ejs

```diff
@@ -710,6 +710,31 @@ var text = mdn.localStringMap({
       'Tools_and_setup': '工具與設定',
       'Design_and_accessibility': '設計與親合度',
     'How_to_contribute': '如何貢獻'
+  },
+  'fr-FR': {
+    'Complete_beginners': 'Les débutants commence ici!',
+    'Getting_started_with_the_web': 'Commencer avec le Web',
+      'Getting_started_with_the_web_overview': 'Vue d\'ensemble de Commencer avec le Web',
+      'Installing_basic_software': 'Installation des outils de base',
+      'What_will_your_website_look_like': 'Quel sera l\'aspect de votre site web ?',
+      'Dealing_with_files': 'Gérer les fichiers',
+      'HMLT_basics': 'Les bases du HTML',
+      'CSS_basics': 'Les bases de CSS',
+      'JavaScript_basics': 'Les bases de JavaScript',
+      'Publishing_your_website': 'Publier votre site web',
+      'How_the_Web_works': 'Le fonctionnement du Web',
+     'HTML_structuring_the_Web': 'HTML',
+      'Introdctiuon_to_HTML': 'Introduction au HTML',
+        'Introduction_to_HTML_overview': 'Introduction au HTML',
+        'Getting_started_with_HTML': 'Commencer avec le HTML',
+        'Whats_in_the_head_metadata_in_HTML': 'Qu\'y-a-t-il dans l\'en-tête ? Métadonnées en HTML',
+        'HTML_text_fundamentals': 'Fondamentaux du texte HTML',
+        'Creating_hyperlinks': 'Création d\'hyperliens',
+        'Advanced_text_formatting': 'Formatage avancé du texte',
+        'Document_and_website_structure': 'Structure de Site Web et de document',
+        'Debugging_HTML': 'Déboguer de l\'HTML',
+        'Assessment_marking_up_a_letter': 'Faire une Lettre',
```

Comment:
"Baliser une lettre" would be closer than "Faire une lettre" which doesn't suggest the "markup" idea of the exercise.

Reply:
Hmm why not, I wasn't sure about that translation, yours is better though!